src/config.ts

```diff
@@ -16,7 +16,7 @@ export const matrixBotPassword = process.env.MATRIX_BOT_PASSWORD as string;
 /** ChatGPT specific stuff */
 export const openAiEmail = process.env.OPENAI_EMAIL as string;
 export const openAiPassword = process.env.OPENAI_PASSWORD as string;
-export const isGoogleLogin = Boolean(process.env.IS_GOOGLE_LOGIN) as boolean;
+export const isGoogleLogin = (process.env.IS_GOOGLE_LOGIN.toLowerCase() === "true") as boolean;
```

Comment:
I think this would throw an exception if `process.env.IS_GOOGLE_LOGIN` is undefined?

Could the line be changed to:

`export const isGoogleLogin = process.env.IS_GOOGLE_LOGIN && process.env.IS_GOOGLE_LOGIN.toLowerCase() === "true";`

Reply:
Yeah It does 

```
TypeError: Cannot read properties of undefined (reading 'toLowerCase')
```

We could just remove the toLowerCase() but it is probably better to have it.

I'll test your suggestion.

Reply:
Yup that works, I'll put in a commit now.

Reply:
Fixed